bin/seeds.rb

```diff
@@ -278,6 +280,15 @@
   "cinder_eqlx_use_chap"          => ['false'],
   "cinder_eqlx_chap_login"        => ['chapadmin'],
   "cinder_eqlx_chap_password"     => [SecureRandom.hex],
+  "cinder_dell_sc_san_ip"           => ['172.23.8.101'],
+  "cinder_dell_sc_san_login"        => ['Admin'],
+  "cinder_dell_sc_san_password"     => [SecureRandom.hex],
```

Comment:
They are enterprise manager login, eventually it will be storage center login when the rest API for storage center is ready.

I've used dell_sc to identify the iscsi driver parameters and dell_fc for fibre channel driver to be consistent